scripts/rebuild_test_artifacts.sh

```diff
@@ -38,4 +38,3 @@ KATANA_RUNNER_BIN=/tmp/katana cargo generate-test-db
 
 # Extracts the database for testing.
 bash ./scripts/extract_test_db.sh
-
```

Comment:
Usually, a blank empty line is appreciated for further insertion, but all good for this one, just nitpicking. :)